### PR TITLE
🌱  Update example version of core CAPI used in clusterctl

### PR DESCRIPTION
**Pull request by killianmuldoon**
Signed-off-by: killianmuldoon <kmuldoon@vmware.com>

Update the version of CAPI used in examples across the clusterctl codebase.

### CODE REVIEW

**Review by ykakarap — APPROVED** [review]
/lgtm

File: cmd/clusterctl/client/delete.go [hunk 1]
```diff
@@ -30,18 +30,18 @@ type DeleteOptions struct {
 	// default rules for kubeconfig discovery will be used.
 	Kubeconfig Kubeconfig
 
-	// CoreProvider version (e.g. cluster-api:v0.3.0) to add to the management cluster. If unspecified, the
+	// CoreProvider version (e.g. cluster-api:v1.1.5) to add to the management cluster. If unspecified, the
```
Comment (enxebre) [inline cmd/clusterctl/client/delete.go:33]: 1.2?

**Comment by sbueringer** [comment]
What is the benefit of trying to keep those versions up-to-date?

File: cmd/clusterctl/client/delete.go (hunk 1 above)
Comment (killianmuldoon) [reply cmd/clusterctl/client/delete.go:33]: I went for this as 1.1.5 is released, recent and stable - I don't think we need the most recent versions possible in these examples (but more recent than 0.3.0 would be nice 🙂

**Comment by killianmuldoon** [comment]
> What is the benefit of trying to keep those versions up-to-date?
I don't think we should be overly zealous about these versions, but something more recent than completely unsupported is probably reasonable. I think these examples are a signal for users and could possibly be used for copy-paste etc. by new users.

**Pull request merged**